packages/theming/atlas/src/themesource/atlas_core/web/design-properties.json

```diff
@@ -254,25 +254,24 @@
             "description": "Change the background color of the container.",
             "options": [
                 {
-                    "name": "Background Default",
+                    "name": "Background Primary",
+                    "oldNames": ["Background Default"],
                     "class": "background-main"
                 },
                 {
-                    "name": "Background Dashboard",
+                    "name": "Background Secondary",
+                    "oldNames": ["Background Dashboard"],
                     "class": "background-secondary"
                 },
-                {
-                    "name": "Background Gradient",
-                    "class": "background-brand-gradient"
-                },
                 {
                     "name": "Brand Primary",
                     "oldNames": ["Primary"],
                     "class": "background-primary"
                 },
                 {
                     "name": "Brand Secondary",
-                    "class": "background-secondary"
+                    "oldNames": ["Default", "Brand Default"],
```

Comment:
Where these design properties used in the past?

Reply:
![image](https://user-images.githubusercontent.com/7356299/112346361-2cebdb80-8cc6-11eb-8e84-82e0484a58c4.png)

Reply:
Yes if you have doubts please check atlas 2 framework. But here it is: https://github.com/mendix/Atlas-UI-Framework/blob/b9af272512cdd860eea686b2156608d4f3bf50cd/settings.json#L202